src/shared/theme/sizes.ts

```diff
@@ -8,11 +10,18 @@ export function sizes(n: number, raw?: boolean) {
 export const zIndex = {
   background: -10,
   farBackground: -20,
+  overlay: 10,
+  nearOverlay: 20,
   header: 100,
   sheetOverlay: 150,
   nearSheetOverlay: 160,
   sideNav: 200,
-  overlay: 10,
-  nearOverlay: 20,
   globalOverlay: 999,
 }
+
+export function square(size: string | number): SerializedStyles {
```

Comment:
nice utility function!
```suggestion
export function square(size: string | number) {
```
but we don't need to specify return type here.